### PR TITLE
fix(test): Make test results more verbose

### PR DESCRIPTION
**Pull request by jarifibrahim**
This PR improves the verbosity of test results using the `jasmine-spec-reporter`[1] module.
**Before**
screenshot from 2017-09-20 23-43-25

**Now**
screenshot from 2017-09-20 23-33-07

[1] - https://github.com/bcaudan/jasmine-spec-reporter

### CODE REVIEW

File: runtime/package.json [hunk 1]
```diff
@@ -110,6 +110,7 @@
     "husky": "0.13.3",
     "istanbul-instrumenter-loader": "2.0.0",
     "jasmine-core": "2.5.2",
+    "jasmine-spec-reporter": "^4.2.1",
```
Comment (vikram-raj) [inline runtime/package.json]: Remove ^.

**Comment by fabric8cd** [comment]
@jarifibrahim fabric8/fabric8-ui:SNAPSHOT-PR-2160-2 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2160-fabric8-planner.badger.fabric8.io

**Comment by rgarg1** [comment]
Good work @jarifibrahim

**Comment by fabric8cd** [comment]
@jarifibrahim fabric8/fabric8-ui:SNAPSHOT-PR-2160-3 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2160-fabric8-planner.badger.fabric8.io

**Comment by fabric8cd** [comment]
@jarifibrahim fabric8/fabric8-ui:SNAPSHOT-PR-2160-4 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2160-fabric8-planner.badger.fabric8.io

**Pull request merged**